source/_docs/configuration/events.markdown

```diff
@@ -45,29 +33,155 @@ Event `call_service` is fired to call a service.
 | `service_data`    | Dictionary with the service call parameters. Example: `{ 'brightness': 120 }`. |
 | `service_call_id` | String with a unique call id. Example: `23123-4`.                              |
 
-## Event `automation_reloaded`
+### `component_loaded`
+
+This event is fired when a new integration has been loaded and initialized.
+
+| Field       | Description                                                                 |
+| ----------- | --------------------------------------------------------------------------- |
+| `component` | Domain of the integration that has just been initialized. Example: `light`. |
 
-Event `automation_reloaded` is fired when automations have been reloaded and thus might have changed.
+### `core_config_updated`
 
-This event has no additional data.
+This event is fired when the core configuration is updated, for example when the location has been changed.
 
-## Event `scene_reloaded`
+## `data_entry_flow_progressed`
 
-Event `scene_reloaded` is fired when scenes have been reloaded and thus might have changed.
+This event is fired when a data entry flow has changed and is used by the frontend to reload the flow state.
 
-## Event `platform_discovered`
+| Field     | Description                 |
+| --------- | --------------------------- |
+| `handler` | The flow handler.           |
+| `flow_id` | Identification of the flow. |
 
-Event `platform_discovered` is fired when a new platform has been discovered by the [`discovery`](/integrations/discovery/) component.
+### `homeassistant_start`, `homeassistant_started`
 
-| Field        | Description                                                                                      |
-| ------------ | ------------------------------------------------------------------------------------------------ |
-| `service`    | The platform that is discovered. Example: `zwave`.                                               |
-| `discovered` | Dictionary containing discovery information. Example: `{ "host": "192.168.1.10", "port": 8889}`. |
+These events are fired during the startup of Home Assistant, in the following order:
 
-## Event `component_loaded`
+1. `homeassistant_start`
+2. `homeassistant_started`
 
-Event `component_loaded` is fired when a new integration has been loaded and initialized.
+These events contain no data.
 
-| Field       | Description                                                                 |
-| ----------- | --------------------------------------------------------------------------- |
-| `component` | Domain of the integration that has just been initialized. Example: `light`. |
+### `homeassistant_stop`, `homeassistant_final_write`, `homeassistant_close`
+
+These events are fired during the shutdown of Home Assistant, in the following order:
+
+1. `homeassistant_stop`
+2. `homeassistant_final_write`
+3. `homeassistant_close`
+
+These events contain no data.
```

Comment:
```suggestion
These events contain no additional data.

Please note that `homeassistant_final_write` and `homeassistant_close`, cannot be used with automations, as the automation engine would already have been stopped when those are fired.

If you want to trigger automation on a Home Assistant stop event, we recommend using the special [Home Assistant trigger](/docs/automation/trigger/#home-assistant-trigger) instead of listening to these events.
```